src/optimizing/Analyze_rule.ml

```diff
@@ -206,13 +206,13 @@ let rec (cnf : Rule.formula -> cnf_step0) =
       And (ys @ zs |> List.concat_map (function And ors -> ors))
   | R.Or (_, xs) ->
       let is_dangerously_large p q =
-        List.compare_length_with p 1_000_000 > 0
-        || List.compare_length_with q 1_000_000 > 0
+        List.compare_length_with p 10_000 > 0
```

Comment:
would have been good to put your PR Comment in the code instead, so one could understand the current values.

Reply:
It's in the commit at least so you can git blame! I was worried it might be too long for the code

Reply:
I d rather avoid git blame.